src/com/projectkorra/projectkorra/waterbending/SurgeWall.java

```diff
@@ -140,19 +139,27 @@ private void freezeThaw() {
 
 	private void freeze() {
 		this.frozen = true;
+
+		tempBlocks.values().forEach(TempBlock::revertBlock);
+		tempBlocks.clear();
```

Comment:
Maybe make a static method in TempBlock called revertBlocks that has one parameter, Collection<TempBlock> that does this? It's already used twice here and I'm sure it could be used in other places including addon abilities

Reply:
Might be too niche though

Reply:
The reason it isn't static is because the instance needs to hold onto what blocks are a part of this particular instance

Reply:
I think you might have missunderstood what I meant.
I mean adding a static method TempBlock#revertBlocks that would have the parameter I was talking about

Reply:
So you would call 
`TempBlock.revertBlocks(tempBlocks);`
`tempBlocks.clear();`

Reply:
I don't quite see the point. A method that is just a loop that reverts all blocks? At most it'd be 3 lines, at least, it'd be 1. It doesn't really optimize anything

Reply:
Yeah that's why I said it might be too niche, I wasn't sure if there were any other places that this was done already